DESCRIPTION

```diff
@@ -26,6 +26,7 @@ Imports:
     mvbutils,
     R6,
     rmarkdown,
+    tools,
```

Comment:
oh wow. I wonder how this got past `R CMD CHECK`?

Reply:
Is it because `tools` is a pre-installed package in R?

Reply:
maybe? I've seen it complain about that before though for `methods` (I thought)

who knows. Either way, good change